preinstall.sh

```diff
@@ -13,6 +13,7 @@ SERVICE_FILE=$(cat <<EOF
 [Unit]
 Description=Viam Services Agent
 After=NetworkManager.service
+After=bluetooth.service
```

Comment:
thinking through consequences of the After

[systemd docs for After directive](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#Before=)

it sounds like this is only respected if 'both units are being started' (i.e. systems without bluetooth will still be able to start agent)

note this has potential to delay startup on systems which have bluetooth -- this is not a merge blocker, but consider pinging SEs before actually releasing this

Reply:
Yes, it used to be "Wants=NetworkManager" and that caused problems for systems without. After means "if you're starting it anyway, sequence."

I did a couple timing tests, bluetooth starts up in basically the same second as NetworkManager on my Pi (same timestamp) so I don't think the delay is an issue. And it's definitely an issue if bluetooth provisioning fails to work because it started up in the wrong order. Will definitely let SE know anyway.